django/university/urls.py

```diff
@@ -9,6 +9,7 @@
     path('course_unit/<int:course_unit_id>/', views.course_unit_by_id),
     path('class/<int:course_unit_id>/', views.classes),
     path('professors/<int:slot>/', views.professor),
-    path('info/', views.info)
+    path('info/', views.info),
+    path('checkHashes/', views.verify_course_units)
```

Comment:
Very good work!

Sorry for the nitpicking, but maybe the endpoint should not contain any implementation det ail (in order for it not being necessary to change the name of the endpoint if for some reason we change the logic inside of the django view).

Maybe `course_unit/validate`?

Reply:
Your suggestion makes sense and as it follows the convention resource/action it's also more meaningful and more consistent with the previous ones